From d360515b1298abb640460bd538b2f3ecb1e55fe9 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Wed, 25 Jul 2018 09:40:10 +0100
Subject: [PATCH 24/26] config/common: Organize filesystems tools

 * Along with it, remove tune2fs as the tool is already part of
   build/target/product/core_minimal.mk. Despite Google did quite
   a big cleanup in master branch, it's still built by default.

Change-Id: I4cf1178c2f99eda5a45de4ba79705093de5cd9bf
---
 config/common.mk | 13 ++++++-------
 1 file changed, 6 insertions(+), 7 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index c7296628..e92e66c5 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -176,20 +176,15 @@ PRODUCT_PACKAGES += \
     bash \
     bzip2 \
     curl \
-    fsck.ntfs \
     gdbserver \
     htop \
     libsepol \
     micro_bench \
-    mke2fs \
-    mkfs.ntfs \
-    mount.ntfs \
     oprofiled \
     pigz \
     powertop \
     sqlite3 \
     strace \
-    tune2fs \
     vim \
     wget
 
@@ -205,10 +200,14 @@ PRODUCT_PACKAGES += \
     libhealthd.lineage
 endif
 
-# exFAT tools
+# Filesystems tools
 PRODUCT_PACKAGES += \
     fsck.exfat \
-    mkfs.exfat
+    fsck.ntfs \
+    mke2fs \
+    mkfs.exfat \
+    mkfs.ntfs \
+    mount.ntfs
 
 # Openssh
 PRODUCT_PACKAGES += \
-- 
2.17.1

